src/MountRenderer.ts

```diff
@@ -111,12 +111,20 @@ export default class MountRenderer implements AbstractMountRenderer {
   }
 
   simulateEvent(node: RSTNode, eventName: string, args: EventDetails = {}) {
-    if (node.nodeType !== 'host') {
-      const name = getDisplayName(node);
-      throw new Error(
-        `Cannot simulate event on "${name}" which is not a DOM element. ` +
-          'Find a DOM element in the output and simulate an event on that.'
-      );
+    let hostNode: Node;
+    if (node.nodeType == 'host') {
+      hostNode = node.instance;
+    } else {
+      let possibleHostNode = nodeToHostNode(node);
```

Comment:
The coding convention in this package is to use `const` unless a binding is re-assigned. We could use a lint rule for this.

Reply:
Will update!

Would you like me to setup eslint for this repo to do this kind of linting?